index.css

```diff
@@ -521,21 +522,13 @@ section ul, section li {
     }
     
     h4 a {
-        font-size: 18px;
-        word-wrap: break-all;
+        word-break: break-all;
```

Comment:
I like the idea of inserting soft hyphens at camel case boundaries. I can add that in a follow-up PR.